### PR TITLE
remove secrun endpoint

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

- remove /secrun endpoint and functionality

## Why are we making these changes?

not used

### CODE REVIEW

File: waiter/src/waiter/process_request.clj [hunk 1]
```diff
@@ -168,9 +168,7 @@
 (defn request->endpoint
```
Comment (shamsimam) [inline waiter/src/waiter/process_request.clj]: Is this function needed anymore. Can we inline the `uri` at its use?

File: waiter/test/waiter/process_request_test.clj [hunk 2]
```diff
@@ -35,40 +35,25 @@
...
-        waiter-headers {(str headers/waiter-header-prefix "endpoint-path") custom-legacy-endpoint}
-        test-endpoints (clojure.set/union legacy-endpoints passthrough-endpoints)]
+  (let [passthrough-endpoints #{"/foo" "/baz/bar" "/load/balancer" "/auto/scale/1/2/3"}
+        waiter-headers {}
+        test-endpoints (clojure.set/union passthrough-endpoints)]
     (doseq [item test-endpoints]
       (testing (str "Test retrieve endpoint with headers and query string: " item)
         (let [dummy-request (assoc (request item :post {:a 1 :b 2}) :query-string "foo=bar&baz=1234")
-              expected-endpoint (if (contains? legacy-endpoints item) custom-legacy-endpoint item)]
+              expected-endpoint item]
```
Comment (shamsimam) [inline waiter/test/waiter/process_request_test.clj]: Maybe rename `item` to `expected-endpoint` in the `doseq` above?

Comment (shamsimam) [reply waiter/test/waiter/process_request_test.clj]: Actually if we inline, we can delete this test :)

File: waiter/src/waiter/process_request.clj (hunk 1 above)
Comment (nsinkov) [reply waiter/src/waiter/process_request.clj]: inlined

**Pull request merged**